app/src/main/java/edu/uc/app/parkfinder/ui/main/HomeFragment.kt

```diff
@@ -20,4 +18,7 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
             findNavController().navigate(action)
         }
     }
+    companion object {
+        fun newInstance() = HomeFragment()
+    }
```

Comment:
Companion object moved to the end

Reply:
This duplicates a previous code review. When that occurs, first in wins.